src/cmd/run.go

```diff
@@ -57,6 +63,22 @@ func doRun(cmd *cobra.Command, args []string) {
 	client.RunnerUnregister(&runner.Id)
 }
 
+func electLeader() {
+	leaseLockName := viper.GetString("runner-deployment")
+	leaseLockNamespace := viper.GetString("runner-pod-namespace")
+	lockIdentity := viper.GetString("runner-pod-name")
+
+	config, err := pkg.GetKubernetesConfig()
+	pkg.CheckErr(err)
+
+	client := clientset.NewForConfigOrDie(config)
+
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
reading this now - this is only created so it can be passed into `pkg.RunLeaderElection` - do we even need it here?  should it just be put into `pkg.RunLeaderElection` ?

Reply:
Yeah, that's a good point.  I'll move it into the function.

Reply:
We're kind of doing the same thing with the k8s client too right?